esteid/bdoc2/ocsp.py

```diff
@@ -0,0 +1,129 @@
+import hashlib
+
+from oscrypto import asymmetric
+
+from ocspbuilder import OCSPRequestBuilder
+from asn1crypto.algos import DigestInfo
+from asn1crypto.ocsp import OCSPResponse, TBSRequestExtension
+from asn1crypto.x509 import Certificate
+
+import requests
+
+from .exceptions import BDoc2Error
+
+
+class OCSPError(BDoc2Error):
+    pass
+
+
+class OCSP(object):
+    """
+    Certificate validation request via the OCSP protocol, using the asn1crypto/ocspbuilder stack.
```

Comment:
s/Certificate/Signature/ ?

Reply:
No, it's OCSP which stands for Certificate Status Protocol, doesn't deal with signatures. Maybe the word `validation` is confusing? Could replace it with `validity confirmation`

Reply:
Oh. What initially confused me was that the third argument for `validate` is `signature`. 

I think this confusion might be avoided if the examples are extended a bit to include both the case where `signature` is provided and the case where it isn't.

Reply:
Yes, that makes sense. The signature argument is used to build a nonce in case the OCSP service is TM-compliant and we'd like to get a TM signature. In the TS case, it's kinda useless. I will add this notice.